doc/en/user/source/production/java.rst

```diff
@@ -3,98 +3,125 @@
 
 .. _java_policyfiles:
 
 Installing Unlimited Strength Jurisdiction Policy Files
--------------------------------------------------------
+```````````````````````````````````````````````````````
 These policy files are needed for unlimited cryptography. As an example, Java does not support AES
-with a key length of 256 bit. Installing the policy files removes these restrictions.
+with a key length of 256 bit.
 
-Open JDK
-````````
 
-Since Open JDK is Open Source, the policy files are already installed.   
+#. Installing the policy files removes these restrictions.
 
-Oracle Java
-```````````
+   * OpenJDK
 
-The policy files are available at   
+     Since Open JDK is Open Source, the policy files are already installed.   
 
-* `Java 8 JCE policy jars <http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html>`__ 
+   * Oracle Java
 
-The download contains two files, **local_policy.jar** and  **US_export_policy.jar**. The default
-versions of these two files are stored in JRE_HOME/lib/security. Replace these two files with the
-versions from the download. 
+     The policy files are available at `Java 8 JCE policy jars <http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html>`__.
 
+     The download contains two files, **local_policy.jar** and  **US_export_policy.jar**. The default ersions of these two files are stored in :file:`JRE_HOME/lib/security`. Replace these two files with the versions from the download. 
 
-Test if unlimited key length is available
-"""""""""""""""""""""""""""""""""""""""""
+   * IBM Java
 
-Start or restart GeoServer and login as administrator. The annotated warning should have disappeared.
+     The policy files are available at `IBM JCE policy jars <https://www14.software.ibm.com/webapp/iwm/web/preLogin.do?source=jcesdk>`__. 
 
-.. figure:: ../security/webadmin/images/unlimitedkey.png
+     An IBM ID is needed to log in. The installation is identical to Oracle.
 
-Additionally, the GeoServer log file should contain the following line::
+#. Test if unlimited key length is available
 
-   "Strong cryptography is available"
+   Start or restart GeoServer and login as administrator. The annotated warning should have disappeared.
 
-.. note::
+   .. figure:: ../security/webadmin/images/unlimitedkey.png
 
-   The replacement has to be done for each update of the Java runtime. 
+#. Additionally, the GeoServer log file should contain the following line::
 
-IBM Java
-````````
+      "Strong cryptography is available"
 
-The policy files are available at
+.. note::
 
-* `IBM JCE policy jars <https://www14.software.ibm.com/webapp/iwm/web/preLogin.do?source=jcesdk>`_ 
+   The installing these policy files needs to be done for each update of the Java runtime. 
 
-An IBM ID is needed to log in. The installation is identical to Oracle.
+Native JAI and ImageIO extensions (not recommended)
```

Comment:
Clear Java 8 subheading for Native JAI and ImageIO, and clearly not recommended.

I considered taking this section to the wiki and removing it from our day-to-day user manual.